src/ast/mod.rs

```diff
@@ -8789,9 +8796,9 @@ mod tests {
     #[test]
     fn test_interval_display() {
         let interval = Expr::Interval(Interval {
-            value: Box::new(Expr::Value(Value::SingleQuotedString(String::from(
-                "123:45.67",
-            )))),
+            value: Box::new(Expr::Value(
```

Comment:
minor stuff we could use the `Expr::value` here too

Reply:
Yes! But I don't have the strength to fight ast-grep again 🙂‍↔️

Reply:
I saw it in my editor and couldn't exist: 
- https://github.com/apache/datafusion-sqlparser-rs/pull/1749